docs/references/production_metrics.md

```diff
@@ -142,4 +146,28 @@ Then you can access the Grafana dashboard at http://localhost:3000.
 
 ### Grafana Dashboard
 
+In a new Grafana setup, ensure that you have the `Prometheus` data source enabled. To check that, go to `http://localhost:3000/connections/datasources` and ensure that `Prometheus` is enabled.
+
+If not, click `Add data source` -> `Prometheus`, set Prometheus URL to `http://localhost:9090`, and click `Save & Test`.
+
 To import the Grafana dashboard, click `+` -> `Import` -> `Upload JSON file` -> `Upload` and select [grafana.json](../examples/monitoring/grafana.json).
+
+### Troubleshooting
+
+#### Check if the variables are created
+
+The example dashboard assume you have the following variables avaliable:
+- `model_name` (name: `model_name`, label: `model name`, Data source: `Prometheus`, Type: `Label values`)
+- `instance` (name: `instance`, label: `instance`, Data source: `Prometheus`, Type: `Label values`)
+
+If you don't have these variables, you can create them manually.
```

Comment:
again, i think there's way to automate this by either grafana iac or making api calls, which we can get them done in a dockerfile for the grafana image, and make it ready-to-use.

creating variables could be even more non-trivial than adding datasource, for users who have no grafana experience. would be really nice to have this automated.

Reply:
In this case I feel it's better to be more explicit and less automated. Setting up variables and seeing previewed values is a also good indication that prometheus is actively receiving metrics data from sglang